WcaOnRails/app/views/registrations/index.html.erb

```diff
@@ -80,4 +89,5 @@
       </tfoot>
     <% end %>
   <% end %>
+  <% end %>
```

Comment:
I know indentation is off, I just wanted to write an easy-to-revert commit even if we try some stuff in the "else" branch.